R/quitteSort.R

```diff
@@ -20,6 +20,6 @@ quitteSort <- function(x) {
   }
 
   x %>%
-    arrange(.data$model, .data$scenario, .data$region, .data$variable,
-            .data$unit, .data$period)
+    arrange(.data$model, .data$scenario, .data$variable,
+            .data$unit, .data$region, .data$period)
```

Comment:
NB: `unit` is superfluous here, as variables can have only one unit by definition.

Reply:
Only as long as you don't merge `GDP (billion US$2010/yr)` and `GDP (billion US$2005/yr)` data into one dataframe. Let us keep it.